packages/linux-kernel/build-jool.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Comment:
This script could do with being marked executable. The other `build-*` in this directory are

Reply:
True, this is one of the things I usually forget to do before commit.